cura/Machines/Models/MachineListModel.py

```diff
@@ -61,7 +61,7 @@ def showCloudPrinters(self) -> bool:
         return self._show_cloud_printers
 
     @pyqtSlot(bool, name="setShowCloudPrinters")
```

Comment:
as name is now the same as the function name the property is redundent 
```suggestion
    @pyqtSlot(bool) 
```